src/es_sfgtools/processing/operations/pride_utils.py

```diff
@@ -96,6 +165,42 @@ def get_daily_rinex_url(date:datetime.date) ->Dict[str,Dict[str,RemoteResource]]
     return urls
 
 def get_gnss_common_products(date:datetime.date) ->Dict[str,Dict[str,Path]]:
+    
+    """
+    Retrieve GNSS common products for a given date.
+    This function fetches various GNSS products (sp3, clk, bias, obx, erp) from 
+    different sources (WuhanIGS and CLSIGS) for the specified date.
+    Args:
+        date (datetime.date): The date for which to retrieve the GNSS products.
+    Returns:
+        Dict[str, Dict[str, Path]]: A dictionary containing the GNSS products 
+        categorized by product type and source. The structure of the dictionary is:
+            {   "sp3": {
+                    "wuhan": Path to WuhanIGS sp3 product,
+                    "cligs": Path to CLSIGS sp3 product,
+                    }.
+                "clk": {
+                    "wuhan": Path to WuhanIGS clk product,
+                    "cligs": Path to CLSIGS clk product,
+                    },
+                "bias": {
+                    "wuhan": Path to WuhanIGS bias product,
+                    "cligs": Path to CLSIGS bias product,
+                    },
+                "obx": {
+                    "wuhan": Path to WuhanIGS obx product,
+                    "cligs": Path to CLSIGS obx product,
+                    },
+                "erp": {
+                    "wuhan": Path to WuhanIGS erp product,
+                    "cligs": Path to CLSIGS erp product,
+                    }
+            }
+    Note:
+        Until the remote resorces are updated with 'update_source', the file_name attribute will be None.
+    """
+    
+
     urls = {
         "sp3": {
             "wuhan": WuhanIGS.get_product_sp3(date),
```

Comment:
is there a reason to try Wuhan before France? my intuition would be to try France then Wuhan, as the download (if they have it) from France should be much faster...

Reply:
good point. I swapped those